config/config.local.minimal.hocon

```diff
@@ -0,0 +1,16 @@
+# The minimum required config options for loading from kinesis
+{
+  "input": {
+    "type": "LocalFs"
```

Comment:
Also very stylistic, but I always found mixing camel case and abbrevations very inconvenient. I.e. why not `LocalFS` and just avoid things like that.

Also I can imagine later adding S3 and GCS, so probably you want to end up with URI format: `file:///tmp/example`

Reply:
I just checked URI format and it is handled already 👍